src/main/resources/js/lgtm-addon.js

```diff
@@ -78,69 +79,68 @@ function loadProjects() {
 }
 
 function handleProjectChange(event) {
-	clearSelect2Field(issueTypeFieldId);
+	lgtmIssueTypeId = null;
 	clearSelect2Field(priorityFieldId);
 	clearSelect2Field(closedStatusFieldId);
 	clearSelect2Field(reopenedStatusFieldId);
 	
 	if (AJS.$(projectFieldId).select2('val') !== "none") {
-		loadIssueTypes(AJS.$(projectFieldId).select2('val'));
+		loadIssueType(AJS.$(projectFieldId).select2('val'));
 		loadPriorities(AJS.$(projectFieldId).select2('val'));
 	}
 }
 
-function loadIssueTypes(projectKey) {
+function loadIssueType(projectKey) {
 	$.ajax({
 		url : AJS.contextPath() + "/rest/api/2/project/" + projectKey + "/statuses"
 	}).done(
 	function(issueTypes) {
-		var fieldContent = issueTypes.map(function(issueType) {
-			return {
-				value : issueType.id,
-				text : issueType.name
-			}
+		var lgtmIssueType = issueTypes.find(function (element) {
+			return element.name === "LGTM Alert";
```

Comment:
`Alert` -> `alert`

Reply:
Jira does not allow creating two issue types with names that only differ in upper/lower case. Therefore I think it would be best to make this check use case-insensitive string comparison.